src/GraphControl/Control/Grapher.cpp

```diff
@@ -699,3 +726,177 @@ namespace GraphControl
         return outputStream;
     }
 }
+
+void Grapher::OnCoreKeyUp(CoreWindow ^ sender, KeyEventArgs ^ e)
+{
+    // We don't want to react to keyboard input unless the graph control has the focus.
+    // NOTE: you can't select the graph control from the mouse for focus but you can tab to it.
+    FrameworkElement ^ whoHasFocus = (FrameworkElement ^) FocusManager::GetFocusedElement();
+    String ^ wName = whoHasFocus->Name;
+    String ^ ETBName = L"GraphingControl";
+    if (wName != ETBName)
+    {
+        return;
+    }
+
+    switch (e->VirtualKey)
+    {
+    case VirtualKey::Left:
+    case VirtualKey::Right:
+    case VirtualKey::Down:
+    case VirtualKey::Up:
+    case VirtualKey::Shift:
+    {
+        HandleKey(false, e->VirtualKey);
+    }
+    break;
+    }
+}
+
+void Grapher::OnCoreKeyDown(CoreWindow ^ sender, KeyEventArgs ^ e)
+{
+    // We don't want to eat keys when the user is in the equation text box.
+    FrameworkElement ^ whoHasFocus = (FrameworkElement ^) FocusManager::GetFocusedElement();
+    String ^ wName = whoHasFocus->Name;
+    String ^ ETBName = L"EquationTextBox";
```

Comment:
Did you mean to keep this in?

Reply:
Fixed, replaced with the same dynamic_cast functionality as keyup